### ISSUE TITLE
Github Flow: I think you are missing controls for git hub flow ?

### ISSUE THREAD

**Issue by Sve1nn** [git-up/GitUp#91]
It would be very nice to see UI options for Github Flow ( Git flow ? ) in GitUp, so far your UI is the best i've seen and it's a shame this feature is missing !

**Comment by mik01aj** [comment]
yes, yes! The perfect version would look like this (upside down of course):

![image](https://cloud.githubusercontent.com/assets/3027415/14018562/c2ace580-f1ce-11e5-84eb-3b4ebf46b3da.png)

**Comment by Sve1nn** [comment]
Thanks for the quick action.. I will be waiting eagerly for this !

**Comment by krosullivan** [comment]
The only thing stopping me from using git flow exclusively.

**Comment by mik01aj** [comment]
I see gitflow as more of a convention than a tool. And I can "do git flow" with GitUp as it is.

On the other hand, I would find one particular addition useful: To always see the master and develop branches on the left. I think that this could be simple to implement (just different branch tip sorting order) and this way these branches would always get the same colors, etc. This could be configurable in repo settings like this:

    Show these branches first: [ master, develop ]
                               Put comma-separated branch names here. They will always be shown in this order.

**Comment by maxkfranz** [comment]
I think the suggestion of @mik01aj makes sense, and it would be useful outside of the git flow usecase besides.

As for having some sort of git flow UI in GitUp, I'm against it.  I use GitUp because it is so minimalistic, it has great keyboard shortcuts, and it doesn't try to constrain or direct my workflow in any way.  Those reasons also make GitUp great for git novices.

There are other tools for which git flow is the focus, and it would be a shame in my opinion to complicate GitUp with git flow UI.

**Comment by kmark** [comment]
Since git-flow is an optional convention on top of git, GitUp would need to function without git-flow anyway. Since git-flow is opt-in I don't believe it'd harm the minimalism of GitUp which could continue to work in the way it always has.

**Comment by mik01aj** [comment]
how about just adding the tiny feature I suggested above? It shouldn't hurt minimalism, and it can help various branch workflows, because then it'd be easier to read the graph of branches.

**Comment by AlexIzh** [comment]
#201 
I added simple git-flow support for gitUp. Currently it doesn't support "support" branches and prefix for tags but seems like another functions work. You can try to use my forked version

**Comment by MichaelMallett** [comment]
If this is to work properly, please please PLEASE use the avh fork of git flow. Git flow has been abandoned as a project for years, and no gui uses the avh release...which is far superior.
